tests/requirements/base.txt

```diff
@@ -6,6 +6,8 @@ jinja2 >= 2.7
 numpy
 Pillow
 PyYAML
+# Remove if it ends up being possible to build libmemcached on the Windows Jenkins box.
```

Comment:
Is there already support for having a different Jenkins config file for the windows job vs the linux one? (So we can have one config with the pylibmc backend, and one without)